components/Navigation/Navigation.js

```diff
@@ -0,0 +1,25 @@
+import { useRouter } from "next/router";
```

Comment:
I think we can remove this as we are not using in this component.

Reply:
Hi Pete! Thanks for your great hints! I removed the unnecessary fragments as well as the router hook. The `dialog`-element we might consider to implement in one of our next US.